lib/ts/recipe/accountlinking/index.ts

```diff
@@ -0,0 +1,136 @@
+            timeJoinedOrder,
+            limit,
+            paginationToken,
+            includeRecipeIds,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async canCreatePrimaryUserId(recipeUserId: string, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.canCreatePrimaryUserId({
+            recipeUserId,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async createPrimaryUser(recipeUserId: string, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.createPrimaryUser({
+            recipeUserId,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async canLinkAccounts(recipeUserId: string, primaryUserId: string, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.canLinkAccounts({
+            recipeUserId,
+            primaryUserId,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async linkAccounts(recipeUserId: string, primaryUserId: string, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.linkAccounts({
+            recipeUserId,
+            primaryUserId,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async unlinkAccounts(recipeUserId: string, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.unlinkAccounts({
+            recipeUserId,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async getUser(userId: string, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.getUser({
+            userId,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async listUsersByAccountInfo(info: AccountInfo, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.listUsersByAccountInfo({
+            info,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async getUserByAccountInfo(info: AccountInfoWithRecipeId, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.getUserByAccountInfo({
+            info,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
+    static async deleteUser(userId: string, removeAllLinkedAccounts: boolean, userContext: any) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.deleteUser({
+            userId,
+            removeAllLinkedAccounts,
+            userContext: userContext === undefined ? {} : userContext,
+        });
+    }
```

Comment:
All these functions should be removed from this file as they are exposed via supertokens/index